iidm/iidm-xml-converter/src/main/java/com/powsybl/iidm/xml/TieLineXml.java

```diff
@@ -45,8 +56,16 @@ private static void writeHalf(TieLine.HalfLine halfLine, NetworkXmlWriterContext
         XmlUtil.writeDouble("b1_" + side, halfLine.getB1(), context.getWriter());
         XmlUtil.writeDouble("g2_" + side, halfLine.getG2(), context.getWriter());
         XmlUtil.writeDouble("b2_" + side, halfLine.getB2(), context.getWriter());
-        XmlUtil.writeDouble("xnodeP_" + side, halfLine.getXnodeP(), context.getWriter());
-        XmlUtil.writeDouble("xnodeQ_" + side, halfLine.getXnodeQ(), context.getWriter());
+        IidmXmlUtil.runUntilMaximumVersion(IidmXmlVersion.V_1_4, context, () -> {
+            XmlUtil.writeDouble("xnodeP_" + side, boundaryPoint.getP(), context.getWriter());
+            XmlUtil.writeDouble("xnodeQ_" + side, boundaryPoint.getQ(), context.getWriter());
+        });
```

Comment:
Same question: I would only write xnodeP and xnodeQ to keep compatibilty with old XIIDM version